src/Amazon.Lambda.Tools/Commands/UpdateFunctionConfigCommand.cs

```diff
@@ -28,6 +28,8 @@ public class UpdateFunctionConfigCommand : LambdaBaseCommand
         {
             LambdaDefinedCommandOptions.ARGUMENT_FUNCTION_NAME,
             LambdaDefinedCommandOptions.ARGUMENT_FUNCTION_DESCRIPTION,
+            LambdaDefinedCommandOptions.ARGUMENT_PACKAGE_TYPE,
```

Comment:
Since package type is not updateable as part of the update config API I don't want to expose that as a switch that would make it look like it could be.

Reply:
Makes sense. Reverted